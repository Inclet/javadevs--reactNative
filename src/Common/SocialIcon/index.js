import React from "react";
import { View, Image } from "react-native";
import themeStyle from '../../Styles/common.style';

const SocialIcon = ({ imageUrl }) => {
  return (
    <View style={themeStyle.socialLoginLabelStyle}>
      <Image source={imageUrl} style={{ width: 32, height: 32 }} />
    </View>
  );
};
export default SocialIcon;
