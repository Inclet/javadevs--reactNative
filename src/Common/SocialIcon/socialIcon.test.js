import React from "react";
import { shallow } from "enzyme";
import SocialIcon from "./index";

describe("SocialIcon", () => {
  let wrapper, props;
  props = {
    imageUrl: {
      url: "../pic/me.jpg"
    }
  };
  wrapper = shallow(<SocialIcon {...props} />);

  it("Should match Snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
