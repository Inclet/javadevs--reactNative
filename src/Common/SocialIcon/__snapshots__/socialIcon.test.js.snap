// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialIcon Should match Snapshot 1`] = `ShallowWrapper {}`;
